style.css

```diff
@@ -0,0 +1,128 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+    background-color: #d3d3d3;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    perspective: 2000px;
+    perspective-origin: 50% 50%;
+}
+.cover{
+    position: absolute;
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+    margin: 0 auto;
+    background-image: url(img/cover.jpg);
+    border-radius: 10px;
+    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
+    transform-origin: left;
+    animation: animate 6s ease-in-out normal forwards;
+    z-index: 1;
+}
+.cover::before{
+    content: '';
+    width: 40px;
+    background: radial-gradient(#575450 6px,transparent 7px) repeat-y;
+    background-size: 50px 30px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    box-sizing: border-box;
+}
+.cover::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: -20px;
+    bottom: 0;
+    width: 60px;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
+    background-size: 50px 30px;
+}
+::-webkit-scrollbar{
+    display: none;
+}
+.paper{
+    position: relative;
+    width: 90%;
+    max-width: 800px;
+    min-width: 400px;
+    height: 480px;
+    margin: 0 auto;
+    background-color: #fafafa;
+    border-radius: 10px;
+    box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.5);
+    overflow: hidden;
+}
+.paper::before{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0px;
+    bottom: 0;
+    width: 40px;
+    background: radial-gradient(#575450 6px, transparent 7px) repeat-y;
+    background-size: 30px 30px;
+    border-right: 3px solid #d44147;
+    box-sizing: border-box;
+    margin-left: 10px;
+}
+.paper::after{
+    content: '';
+    width: 60px;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
+    background-size: 50px 30px;
+    position: absolute;
+    top: 0;
+    left: -20px;
+    bottom: 0;
+}
+.paper-content{
+    position: absolute;
+    top: 30px;
+    right: 0px;
+    bottom: 30px;
+    left: 50px;
+    background: linear-gradient(transparent, transparent 28px, #91d1d3 28px);
+    background-size: 30px 30px;
+}
+.paper-content textarea{
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
+    line-height: 30px;
+    background: transparent;
+    padding: 0 10px;
+    border: 0;
+    resize: none;
+    outline: none;
```

Comment:
What does `resize` and `outline` do ??
@murtazshaikh

Reply:
resize :- In <textarea> by default we can resize the text-area, so to maker it non-resizable `resize: none;'  property is used.
outline :- whenever we start typing outline comes, so to remove it we use `outline: none' property.